__tests__/lint/assertions-custom-function-options/.redocly.yaml

```diff
@@ -0,0 +1,16 @@
+apis:
+  main:
+    root: ./openapi.yaml
+styleguide:
+  plugins:
+    - ./plugin.js
+  rules:
+    assert/operation-summary-length:
+      subject: Operation
+      property: summary
+      message: Operation summary should be longer
+      function:
+        name: local/checkLength
+        options: 
+          min: 10
```

Comment:
I lean towards this syntax:

```yaml
local/checkLength:
  min: 10
```

It will allow us to put more than one validation function into an assertion. But I'm not sure. 
@adamaltman , @RomanHotsiy , what do you think?

Reply:
I agree @tatomyr although I'm not sure how practical more than one in an assertion is, it may turn out to be practical.